dashboard/src/actions/apps.test.tsx

```diff
@@ -363,16 +363,20 @@ describe("rollbackApp", () => {
 
   it("success and re-request apps info", async () => {
     App.rollback = jest.fn().mockImplementationOnce(() => true);
+    App.getRelease = jest.fn().mockImplementationOnce(() => true);
     const res = await store.dispatch(provisionCMD);
     expect(res).toBe(true);
 
     const expectedActions = [
       { type: getType(actions.apps.requestRollbackApp) },
       { type: getType(actions.apps.receiveRollbackApp) },
       { type: getType(actions.apps.requestApps) },
+      { type: getType(actions.apps.selectApp), payload: true },
     ];
     expect(store.getActions()).toEqual(expectedActions);
     expect(App.rollback).toHaveBeenCalledWith("my-release", "default", 1);
+    // It should re-get the app after the rollback, note that the parameters are switched
```

Comment:
Can we instead update `App.getRelease` (and call-sites) to be consistent with all the other signatures in `shared/App.ts`?

Reply:
actually the "problem" is `getAppWithUpdateInfo` and `getRelease`, all the other functions has the `releaseName` before the `namespace`. I didn't want to change the functions though because it was riskier... Anyway, will try

Reply:
well, actually no, `getAppWithUpdateInfo` is used in many places along with other functions that use the `namespace, name` structure which I believe is better because is closer to an API call in Kubernetes, we would need to change all App.* functions to use that approach instead... still quite risky

Reply:
But it has to be done, no? I'll +1 this so you can land it if you want, but I don't understand why we wouldn't fix it to be consistent now and avoid confusion / risk in the future.

Reply:
yes, I just didn't have the time yesterday to complete this, updating this